src/blocks/Footer/index.js

```diff
@@ -34,7 +34,7 @@ const LogoSection = styled.div`
 `
 
 const Footer = props => {
-  const footerProps = getFooterData()
+  const footerProps = footerData
```

Comment:
Can you rework this to remove the `footerProps` useless variable ? :)

Reply:
Yes, I'll get on it now